tests/Tinkoff.Trading.OpenApi.Tests/Tinkoff.Trading.OpenApi.Tests.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFramework>net5.0</TargetFramework>
```

Лучше прогонять тесты сразу на 3.1 и на 5.0
```suggestion
        <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
```